app/controllers/callbacks_controller.rb

```diff
@@ -2,7 +2,6 @@
 
 class CallbacksController < Devise::OmniauthCallbacksController
   def github
-    # binding.pry
```

Comment:
thanks for removing it - please not that the commit message doesn't match the change (fine for now, no need to change, just a remark for the future)